lib/concord/utils.rb

```diff
@@ -1,13 +1,17 @@
-module Concord
-  # Some helper functions useful in client computations
-  module Utils
-    extend self
+# Some helper functions useful in client computations
+module Concord::Utils
+  extend self
 
-    # Get the time in milliseconds
-    # @param t [Time] The time to convert to milliseconds
-    def time_millis(t = Time.now)
-      (t.to_f * 1000.0).to_i
-    end
+  # Get the time in milliseconds
+  # @param t [Time] The time to convert to milliseconds
+  def time_millis(t = Time.now)
+    (t.to_f * 1000.0).to_i
   end
-end
 
+  # Log message to stderr
+  # @param str [String] The message to print to stderr
+  def log(str)
```

Comment:
Note: @senior7515 commented in getting_started repo that this should be named log_to_stderr